### PR TITLE
kev/341_delete_helper_rename_util_functions

### PR DESCRIPTION
**Pull request by junhaow1**
# Pull Request Details

## What issue does this PR address

- delete helper.dart file and rename util functions

- Link to associated issue: #341 

## Checklist

Complete the check-list below to ensure your branch is ready for PR.

Flutter Style Guide: https://survivor.togaware.com/gnulinux/flutter-style.html

- [x] Screenshots included in linked issue
- [x] Changes adhere to the style and coding guideline
- [x] No confidential information
- [x] No duplicated content
- [x] No lint check errors related to your changes (`make prep` or `flutter analyze lib`)
- [ ] Pre-exisiting lint errors noted: [HERE]
- [x] Tested on at least one device
  - [ ] Android Phone
  - [ ] Android Emulator
  - [ ] Chrome on Android
  - [ ] Chrome
  - [ ] iOS
  - [x] Linux
  - [x] MacOS
  - [ ] Windows
- [ ] Added a reviewer

## Finalising

Once PR discussion is complete and reviewer has approved:

- [ ] Merge dev into the branch
- [ ] Resolve any conflicts
- [ ] Add one line summary into CHANGELOG.md
- [ ] Bump appropriate version number in pubspec.yaml
- [ ] Push to git repository and review
- [ ] Merge PR into dev (gjwgit)

### CODE REVIEW

**Comment by junhaow1** [comment]
To keep this PR focus on renaming the functions, next PR will focus on impute testing alone

File: integration_test/utils/navigate_to_tab.dart
Comment (gjwgit) [inline integration_test/utils/navigate_to_tab.dart:1]: Lots of repeated code here. DRY. Couldn't you have a structure like [{'tab': 'Explore', 'icon': Icons.insights, 'type': ExploreTabs}, ...] (as a first guess) and have a single body, and be sure to cover all tabs?

**Review by gjwgit — CHANGES_REQUESTED** [review]

File: integration_test/utils/navigate_to_tab.dart
Comment (junhaow1) [reply integration_test/utils/navigate_to_tab.dart:1]: on it

Comment (junhaow1) [reply integration_test/utils/navigate_to_tab.dart:1]: eliminates repeated if-else conditions : 
image

**Review by gjwgit — APPROVED** [review]
Nicely done. Thanks @junhaow1.

I added some pauses for timing of the itest.

**Review by zheyxu — APPROVED** [review]

**Pull request merged**